src/utils.js

```diff
@@ -57,27 +57,25 @@ function getEnvironmentVariables(twilioClient, shouldGetEnvFromFile) {
  * Execute Pulumi CLI command
  *
  * @param {Array} args Arguments to Pulumi CLI command
- * @param {boolean=true} interactive Whether to run the command in interactive mode (i.e. gathering input from user)
+ * @param {boolean} checkEnvFile Whether to search and load env file for the current stack 
+ * @param {boolean=true} interactive Whether to run the command in interactive mode (i.e. gathering input from user) 
  * @param {{}} twilioClient Initialized Twilio client
  * @return {null | string} Pulumi command output if executed non interactively. Void if execute interactively
  */
 
-function runPulumiCommand(args, interactive = true, twilioClient) {
+function runPulumiCommand(args, checkEnvFile, interactive = true, twilioClient) {
```

Comment:
@kaiquelupo is there a better way to implement this without adding an argument to the `runPulumiCommand()`? I'd rather handle all the _internals_ of Pulumi CLI commands within the  `runPulumiCommand()`. We should see this function as adaption layer that abstract third party tool, so in the future we can add / replace pulumi with other provider as well.

Reply:
Got it, makes sense. The other way would be adding 

`    const shouldGetEnvFromFile = 
      (args[0] !== "stack" && args[1] !== "ls") && 
      (args[0] !== "new");`

to the runPulumiCommand and return everything else to how they were.

My only concern is that we are listing the commands we don't want to seach for the env file  and I am not sure if this approach is better then the one on this PR right now. What do you think @vernig ?

Reply:
Yes, I think this is a more straightforward implementation because it requires changes in one single place. So if in the future we want to change this implementation, we don't have to change all the places where we call the `runPulumiCommand()`.

Reply:
Right! I will make the change now